### ISSUE TITLE
[Feature Request] Store & extract original URL for document

### ISSUE THREAD

**Issue by setr** [burtonator/polar-bookshelf#294]
the PHZ files at least don't appear to be specifically storing the original URL at this time.

Storing it in general would probably be useful for automatically re-capturing documents for psuedo-backwards compatibility (eg store youtube links now, and re-capture it when video support improves; Or even if that would be a UI-only change, then things like not capturing youtube's infinite-loading recommendations). 

Extracting it would be useful for sharing documents/links

**Comment by burtonator** [comment]
> the PHZ files at least don't appear to be specifically storing the original URL at this time.

Why do you say that?  They should be.  

I don't totally follow this feature request. Maybe rephrase?

**Comment by setr** [comment]
Didn’t see anything specifically storing it on a superficial scan of an
archive; first part of the request is irrelevant if its already being
stored.

By extract I just mean getting the url out of a document ie by
right-clicking, to paste somewhere else (ie IM’ing someone the article, or
posting on HN or something).
